parsers/UY.py

```diff
@@ -69,13 +94,17 @@ def parse_page(session):
     # If this assumption is not correct, then the following line should be uncommented.
     # obj['Hidráulica'] -= max(0, obj['Interconexión Salto Grande'])
```

Comment:
We should probably explain why we ignore `obj['Interconexión Salto Grande']` and use `http://www.cammesa.com/uflujpot.nsf/FlujoW?OpenAgent&Tensiones y Flujos de Potencia&` instead